adagrams/game.py

```diff
@@ -1,13 +1,126 @@
 from random import randint
 
 def draw_letters():
-    pass
+    tiles = [
+    'A', 'A', 'A', 'A', 'A', 'A', 'A', 'A', 'A', 
+    'B', 'B',
+    'C', 'C',
+    'D', 'D', 'D', 'D', 
+    'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 
+    'F', 'F', 
+    'G', 'G', 'G', 
+    'H', 'H',  
+    'I', 'I', 'I', 'I', 'I', 'I', 'I', 'I', 'I', 
+    'J',  
+    'K', 
+    'L', 'L', 'L', 'L', 
+    'M', 'M', 
+    'N', 'N', 'N', 'N', 'N', 'N', 
+    'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 
+    'P', 'P', 
+    'Q', 
+    'R', 'R', 'R', 'R', 'R', 'R', 
+    'S', 'S', 'S', 'S', 
+    'T', 'T', 'T', 'T', 'T', 'T',
+    'U', 'U', 'U', 'U', 
+    'V', 'V', 
+    'W', 'W', 
+    'X', 
+    'Y', 'Y', 
+    'Z'
+    ]
+    selected_index = []
+    picked_letters = []
+    for i in range(0, 10):
+        rand_selection = randint(0, len(tiles)-1)
```

Comment:
I'd prefer a more descriptive name for this variable to make your code more self-documenting. Maybe like `random_index`

```suggestion
        random_index = randint(0, len(tiles)-1)
```